adafruit_pcf8591/analog_in.py

```diff
@@ -27,11 +27,16 @@
 * Author(s): Bryan Siepert, adpted from ADS1x15 by Carter Nelson
 """
 
+try:
+    import typing  # pylint: disable=unused-import
+except ImportError:
+    pass
+
 
 class AnalogIn:
     """AnalogIn Mock Implementation for ADC Reads."""
 
-    def __init__(self, pcf, pin):
+    def __init__(self, pcf: "PCF8591", pin: int) -> None:
```

Comment:
I think you should be able to actually import `adafruit_pcf8591.pcf8591.PCF8591` in the typing block - I don't think it's a cyclic import so it shouldn't be a problem.  Then you won't need to use quotes.